src/query/storages/memory/src/memory_table.rs

```diff
@@ -233,7 +233,12 @@ impl Table for MemoryTable {
         _: AppendMode,
         _: bool,
     ) -> Result<()> {
-        pipeline.add_sink(|input| Ok(ContextSink::create(input, ctx.clone())))
+        pipeline.add_sink(|input| {
+            Ok(ProcessorPtr::create(ContextSink::create(
```

Comment:
Move `ProcessorPtr::create` wrap to `add_sink` or `ContextSink::create` internal? 
The intention of doing a wrap here is not too clear.

Reply:
Ditto.